lib/domain/model/room/room_extension.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'package:fluffychat/domain/model/search/search_model.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/client_stories_extension.dart';
+import 'package:matrix/matrix.dart';
+
+extension RoomExtension on Room {
```

Comment:
please use anotherName, more specific, because there are RoomExtension in utils folder

Reply:
Don't worry, it's only one `RoomExtension `